test/types/model/Scan.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint @typescript-eslint/no-unused-vars: 0 */
+
+import {UserTypedModel} from "../Model";
+
+const shouldScanSuccessfully = UserTypedModel.scan("name").eq("john").exec();
```

Comment:
Shouldn't we add tests for the `lt`, `le`, etc. methods here?

Reply:
added more tests